CRM/Utils/SQL/Insert.php

```diff
@@ -65,7 +65,7 @@ public static function into($table, string $verb = 'INSERT INTO') {
    * @throws \CRM_Core_Exception
    */
   public static function dao(CRM_Core_DAO $dao) {
-    $table = CRM_Core_DAO::getLocaleTableName($dao->getTableName());
+    $table = $dao::getLocaleTableName();
```

Comment:
'm pretty sure this had been localizing the table name twice, because `getTableName` was already calling `getLocaleTableName()` internally. This would have been harmless because the 2nd pass would have done nothing, but it was silly and confusing, like much of our localization code.